install-dev/upgrade/sql/1.7.3.0.sql

```diff
@@ -3,3 +3,16 @@ SET NAMES 'utf8';
 
 UPDATE `PREFIX_tab` SET `position` = 0 WHERE `class_name` = 'AdminZones' AND `position` = '1';
 UPDATE `PREFIX_tab` SET `position` = 1 WHERE `class_name` = 'AdminCountries' AND `position` = '0';
+
+CREATE TABLE IF NOT EXISTS `PREFIX_store_lang` (
+  `id_store` int(11) unsigned NOT NULL,
+  `id_lang` int(11) unsigned NOT NULL,
+  `name` varchar(128) NOT NULL,
+  `address1` varchar(128) NOT NULL,
+  `address2` varchar(128) DEFAULT NULL,
+  `hours` text,
+  `note` text,
+  PRIMARY KEY (`id_store`, `id_lang`)
+) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
```

Comment:
Actually I don't believe that `COLLATION` is valid SQL, I think it's only a constant that's replaced by a preprocessor, but I'm not sure if that's parsed on this file.
Could you please check? Because if the preprocessor doesn't replace it, the query will be invalid and fail.